src/App.tsx

```diff
@@ -1,109 +1,39 @@
 import React from 'react';
-
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+import { Post } from './types/Post';
+import { PostList } from './components/PostList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const getComments = (postId: number): Comment[] => {
+  const foundComments = commentsFromServer.filter(
+    comment => comment.postId === postId,
+  );
+
+  return foundComments;
+};
```

Comment:
Why do you use function declaration and arrow function? Let's keep same style (choose one).